### ISSUE TITLE
textPath refering to path with transform

### ISSUE THREAD

**Issue by JoKalliauer** [RazrFalcon/resvg#270]
#### xml-code
```xml
<?xml version="1.0" encoding="UTF-8"?>
<svg width="640" height="640" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" font-family="Liberation Sans,Arial" font-size="14" text-anchor="middle">
 <circle cx="320" cy="320" r="212.5" fill="none" stroke="#ff0" stroke-width="15"/>
 <circle cx="320" cy="320" r="232.5" fill="none" stroke="#ff0" stroke-width="15"/>
 <circle cx="320" cy="320" r="252.5" fill="none" stroke="#ff0" stroke-width="15"/>
 <circle cx="320" cy="320" r="272.5" fill="none" stroke="#ff0" stroke-width="15"/>
 <circle cx="320" cy="320" r="292.5" fill="none" stroke="#ff0" stroke-width="15"/>
 <circle cx="320" cy="320" r="312.5" fill="none" stroke="#ff0" stroke-width="15"/>
  <path id="a" transform="translate(920 -1332.4)" d="m-700.04 1377.5a292.5 292.5 0 0 1 200.08 0" fill-opacity="0.2"/>
  <text transform="translate(0 25)" y="20" ><textPath startOffset="50%" xlink:href="#a">1+3+5+7+(9)</textPath></text>
  <text transform="translate(0 5)" y="20" ><textPath startOffset="50%" xlink:href="#a">1+3+5+(7+9+11)</textPath></text>
  <text transform="translate(0 45)" y="20" ><textPath startOffset="50%" xlink:href="#a">1+3+5+7</textPath></text>
  <text transform="translate(0 65)" y="20" ><textPath startOffset="50%" xlink:href="#a">1+3+5</textPath></text>
  <text transform="translate(0 85)" y="20" ><textPath startOffset="50%" xlink:href="#a">1+3</textPath></text>
  <text transform="translate(0 -15)" y="-20" ><textPath startOffset="50%" xlink:href="#a">1+3+(5+7+9+11+13)</textPath></text>
</svg>
```

#### What happened?
|lib|pic|
|-|-|
|svg|![](https://upload.wikimedia.org/wikipedia/commons/d/d1/Uranium_atom.svg)
|ink|![UraniumsO4S4nkssS0_vOS](https://user-images.githubusercontent.com/15687581/76104067-0253b580-5fd3-11ea-9974-fc18fa848561.png)
|resvg|![UraniumsO4S4nkssS0_vOS_re](https://user-images.githubusercontent.com/15687581/76104082-07b10000-5fd3-11ea-9a5d-f4f70533642d.png)
|rsvg|![UraniumsO4S4nkssS0_vOS_r247](https://user-images.githubusercontent.com/15687581/76104099-0b448700-5fd3-11ea-86b8-277aeb78e695.png)

![Screenshot from 2020-03-06 17-51-52](https://user-images.githubusercontent.com/15687581/76104184-2dd6a000-5fd3-11ea-9b0f-1d88f4c97f23.png)

#### What should have happened?
 - textPath not visible on screen

#### Version and Operating System:

- Inkscape Version: 0.92.4 or 1.1dev
- resvg version: 0.9.0
- rsvg Version: 2.40.21 or 2.47.3
- Operating System: Ubuntu 18.04

#### Source
https://commons.wikimedia.org/wiki/File:Uranium_atom.svg

**Comment by Stoeoef** [comment]
Should be fixed with #292 
![Out](https://user-images.githubusercontent.com/5731969/83573834-7d722c80-a52c-11ea-9d40-f5618657151c.png)

**Comment by JoKalliauer** [comment]
Thanks to @Stoeoef . I can confirm it is issue is fixed.
